### PR TITLE
generator for system spec

### PR DESCRIPTION
**Pull request by andrzejsliwa**
Rspec since 3.7 is supporting the system specs, so would be great to add generator for it.

### CODE REVIEW

**Comment by JonRowe** [comment]
I think you need to make sure these specs only run on rails versions that support system specs.

**Comment by joshRpowell** [comment]
@andrzejsliwa #2071 looks like a good candidate for including this PR

**Comment by JonRowe** [comment]
No this can be independant, it just needs to be gated on Rails version

File: lib/generators/rspec/system/system_generator.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+require 'generators/rspec'
+
+module Rspec
+  module Generators
+    # @private
+    class SystemGenerator < Base
+      class_option :system_specs, :type => :boolean, :default => true,  :desc => "Generate system specs"
+
+      def generate_feature_spec
+        return unless options[:system_specs]
```
Comment (joshRpowell) [inline lib/generators/rspec/system/system_generator.rb]:
```ruby
@andrzejsliwa perhaps you'll have something more elegant in mind, but this is a start:
begin
  raise ArgumentError.new('System Tests are available for Rails >= 5.1') if Rails::VERSION::STRING < '5.1'
rescue ArgumentError => e
  puts e.message
end
```

**Comment by joshRpowell** [comment]
> No this can be independant, it just needs to be gated on Rails version

Thanks @JonRowe. Dropped a something to jumpstart the conversation again.

**Comment by benoittgt** [comment]
Closed via: https://github.com/rspec/rspec-rails/pull/2075 thanks @andrzejsliwa and @joshRpowell 🎉

**Pull request closed without merging**